glide.lock

```diff
@@ -169,60 +138,27 @@ imports:
   version: e00690e87603abe613e9f02c816c7c4bef82e063
   subpackages:
   - openstack
-  - openstack/blockstorage/v1/volumes
```

Comment:
Probably mostly my fault when I was poking at glide trying to get the apiserver to build initially.